server/adaptors/integrations/__data__/repository/aws_vpc_flow/aws_vpc_flow-1.0.0.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "aws_vpc_flow",
+  "version": "1.0.0",
+  "displayName": "AWS VPC Flow",
+  "description": "AWS VPC Flow log collector",
+  "license": "Apache-2.0",
+  "type": "logs_vpc",
+  "author": "Haidong Wang",
+  "sourceUrl": "https://github.com/opensearch-project/observability/tree/2.x/integrations/aws_vpc_flow_log",
```

Comment:
getting a 404, can we update this, let's stop pointing to observability backend repo if possible

Reply:
we do need this since all our docs are over there (Observability Repo)